main/EDPftvarcon.F90

```diff
@@ -134,20 +134,26 @@ module EDPftvarcon
      ! Nutrient Aquisition (ECA & RD)
 
 
-     !real(r8), allocatable :: rd_vmax_n(:)             ! maximum production rate for plant n uptake   [gN/gC/s]
      real(r8), allocatable :: decompmicc(:)             ! microbial decomposer biomass gC/m3
                                                         ! on root surface
 
+     real(r8), allocatable :: vmax_nh4(:) ! maximum production rate for plant N uptake
+                                          ! For ECA: this is just ammonium: nh4 uptake   [gN/gC/s]
+                                          ! For RD: this is the uptake of both (we can't have
+                                          ! unique parameter for RD, because it want's one demand,
+                                          ! and uses this vmax to set the demand, first drawing from
+                                          ! NH4 and then NO3
+     
      ! ECA Parameters: See Zhu et al. Multiple soil nutrient competition between plants,
      !                     microbes, and mineral surfaces: model development, parameterization,
      !                     and example applications in several tropical forests.  Biogeosciences,
      !                     13, pp.341-363, 2016.
      ! KM: Michaeles-Menten half-saturation constants for ECA (plant–enzyme affinity)
      ! VMAX: Product of the reaction-rate and enzyme abundance for each PFT in ECA
      ! Note*: units of [gC] is grams carbon of fine-root
-
+     
      real(r8), allocatable :: eca_km_nh4(:)   ! half-saturation constant for plant nh4 uptake  [gN/m3]
-     real(r8), allocatable :: eca_vmax_nh4(:) ! maximum production rate for plant nh4 uptake   [gN/gC/s]
+     
```

Comment:
this is because the vmax_nh4 parameter is used for both eca and rd, while the vmax_no3 parameter is just used for eca.  Again, it might be better to use both to define demand for RD, as a sum.